crypto/store/loader_file.c

```diff
@@ -14,16 +14,16 @@
 #include <assert.h>
 
 #include <openssl/bio.h>
-#include <openssl/dsa.h>         /* For d2i_DSAPrivateKey */
+#include <openssl/dsa.h> /* For d2i_DSAPrivateKey */
 #include <openssl/err.h>
 #include <openssl/evp.h>
 #include <openssl/pem.h>
-#include <openssl/pkcs12.h>      /* For the PKCS8 stuff o.O */
-#include <openssl/rsa.h>         /* For d2i_RSAPrivateKey */
+#include <openssl/pkcs12.h> /* For the PKCS8 stuff o.O */
+#include <openssl/rsa.h> /* For d2i_RSAPrivateKey */
```

Comment:
Why?

Reply:
If this was for check-format, then I think check-format is asking too much.  In the original source-format stuff, we actually have column 33 as the starting column for these kinds of same-line one-line comments.

Reply:
Well, as I wrote this is just to satisfy the checks of `check-format.pl`, which otherwise report "double space" at least if there is just one such comment not directly following code.

Actually for the two lines highlighted above the tool would not have reported this because the comments in two consecutive lines started on the same column.
Shall I revert this?

Reply:
Ah, I just got your second comment.
So I've changed back the for first line to comments starting at column 33.